sdcm/tester.py

```diff
@@ -2953,10 +2953,21 @@ def clean_resources(self):
 
         self.destroy_credentials()
 
+    def save_nodes_schema_before_test_finish(self):
```

Comment:
We may want to call it at any point of a test run. For example - before decommission nemesis.
So, do not include current final goal in the naming.
Include only actual action in the naming.